### ISSUE TITLE
[Feature Request] Enriched Biological Ecosystem

### ISSUE THREAD

**Issue by ghost** [Regalis11/Barotrauma#1383]
## Based on [this post](https://steamcommunity.com/app/602960/discussions/0/1850323802589812744/) in the steam community.

Enriching the Europa wild life past the point of kill or be killed by the player - but perhaps add herbivores and simulate some sort of behaviour that does not affect the player directly, but simply add him to the food chain some sorts?

A reference to well implemented concept of this would be rainworld.

- [x] I have searched the issue tracker to check if the feature has already been requested.

**Comment by AbominationBanisher** [comment]
Haha, you searched the github but you didn't search the undertow forums! Sorry rookie, you'll get used to it eventually >:).

https://undertowgames.com/forum/viewtopic.php?f=19&t=5969&p=114962&hilit=ecosystem#p114962

In fact, Regalis even commented on that exact post. It even has that Rainworld reference and everything in it, primarily because as complicated as it may seem, I saw that post on the undertow forums two years ago, saw your post on steam two days ago, commented about rainworld on the exact steam discussion you referenced, and now you included rainworld in this post. A prime example of a chain reaction.

tldr; already said and thought about

**Comment by Regalis11** [comment]
We're moving feature requests from the issue tracker to the Barotrauma forums. From now on only actual issues (bugs, crashes, design flaws, oversights) should be reported here. We're doing this because the issue tracker has started to become extremely cluttered - it's getting harder for us to find the more critical issues when they're mixed in with hundreds of ideas and feature requests. And even though we are closing the existing feature requests, we are still planning on implementing many of them further down the line. 

So from now on, please post your feature requests on the [Collaboration section on the Barotrauma forums](https://undertowgames.com/forum/viewforum.php?f=19&sid=3ef7ed701ca460dc3503e2ab4e3cf7c9).
